src/_internal/selection/src/styles/index.cssr.ts

```diff
@@ -113,7 +114,9 @@ export default c([
       position: relative;
       z-index: auto;
       display: flex;
-      padding: var(--n-padding-single);
+      padding-top: 3px;
```

Comment:
If we anyway introduce a new variable, would it be possible to make this themable at the same time?